.github/workflows/node-pr-jobs-secure.yml

```diff
@@ -47,13 +47,14 @@ jobs:
       - name: Test client code
         run: npm run test:jest:client -- --forceExit # force exit in case a test does not clean up after failure
       - name: Test server code
+        if: ${{ success() || failure() }}
         run: npm run test:jest:server -- --forceExit # force exit in case a test does not clean up after failure
       - name: Coverage report
         id: coverage
-        if: ${{ always() }}
+        if: ${{ success() || failure() }}
```

Comment:
Looking at the code, yep. It'll fail the step if it can't read the coverage but that would be a bug regardless of this change. Worth raising an issue to handle that a bit better since we should be showing whatever coverage we do have.